com.one_ware.OneWare.yml

```diff
@@ -0,0 +1,49 @@
+app-id: com.one_ware.OneWare
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet8
+build-options:
+  prepend-path: "/usr/lib/sdk/dotnet8/bin"
+  append-ld-library-path: "/usr/lib/sdk/dotnet8/lib"
+  prepend-pkg-config-path: "/usr/lib/sdk/dotnet8/lib/pkgconfig"
+  arch:
+    aarch64:
+      env:
+        RUNTIME: "linux-arm64"
+    x86_64:
+      env:
+        RUNTIME: "linux-x64"
+
+command: oneware
+separate-locales: false
+tags:
+  - proprietary
```

Comment:
```suggestion
```

Reply:
ah I missed this one, I changed it in the metainfo